src/stylesheets/components/_alerts.scss

```diff
@@ -74,25 +106,13 @@ $alert-padding-left: units($theme-alert-padding-x) +
 }
 
 .usa-alert__text:only-child {
-  margin-bottom: units($theme-alert-bar-width);
-  padding-top: units(0.5);
-}
-
-@each $name, $icon in $alert-icons {
-  .usa-alert--#{$name} {
-    @include alert-status-styles($name, $icon);
-  }
+  @include u-padding-y(0);
 }
 
 .usa-alert--slim {
   @include alert-slim-styles;
 }
 
-.usa-alert--no-heading {
-  background-position: $alert-padding-left
-    calc(#{units($theme-alert-padding-x)} - #{units(0.5)});
-}
-
 .usa-alert--validation {
```

Comment:
This variant needs the latest changes.

![image](https://user-images.githubusercontent.com/3385219/110993457-f06ec600-833c-11eb-9a2c-6865b8098bfa.png)
[Source](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/dw-color-icons/components/detail/validation.html)

Reply:
DONE